targets/XPlatCppSdk/source/code/include/playfab/PlayFabTransportHeaders.h

```diff
@@ -1,17 +1,17 @@
 #pragma once
 
-#ifdef _DURANGO
+#include <playfab/PlayFabPlatformMacros.h>
+
+#ifdef PLAYFAB_PLATFORM_XBOX
 #include <playfab/PlayFabIXHR2HttpPlugin.h>
-#endif // _DURANGO
+#endif // PLAYFAB_PLATFORM_XBOX
 
-#ifdef __linux__
+#ifdef PLAYFAB_PLATFORM_LINUX
 #include <curl/curl.h>
 #include <playfab/PlayFabHttp.h>
```

Comment:
These seem to be the same for Linux and Windows. Can we unite those under one condition?